app/src/features/FhirResourceTree/TreeItemLabel.tsx

```diff
@@ -159,44 +154,60 @@ const TreeItemLabel = ({
       >
         {elementNode.type}
       </Typography>
-      <TreeNodeBadge elementNode={elementNode} />
-      {isArrayItem && (
-        <IconButton icon={IconNames.TRASH} onClick={handleDeleteItemClick} />
+      {elementNode.kind === "complex" && !elementNode.isArray && (
+        <Tooltip title={t("addExtension").toString()} arrow>
```

Comment:
Does typescript raise an type error or something ? 🤨

Reply:
Yes !
```
Type 'TFunctionResult' is not assignable to type 'boolean | ReactChild | ReactFragment | ReactPortal'.
 Type 'undefined' is not assignable to type 'boolean | ReactChild | ReactFragment | ReactPortal'.
```

Reply:
`t<string>("addExtension")` seems to work as well. no preference though..